### ISSUE TITLE
Thunderbird email signup

### ISSUE THREAD

**Issue by ScottDowne** [mozilla/donate.mozilla.org#1368]
We need to figure out what needs to happen for this.

From discussing with a few people, I *think* the best way forward is this.

Put thunderbird signups in exact target as a place to store the emails. We don't need to send any emails from exact target, if we don't want to. From here we can figure out what to do with them.

This unblocks us, and we can improve later. We can import then to something else or integrate another mail service or automate the importing process, or something.

cc @edrushka @alicoding @davidascher @vazquez

**Comment by edrushka** [comment]
Yes, this was how I imagined this would work. Can you report back on:
1) How can we mark TB-specific donors in ET to create a special group?
2) Confirm that we can export a list of names/emails from this group.

**Comment by davidascher** [comment]
3) confirm rkent can import said list into mailchimp and email without
mailchimp managing the double opt in.

**Comment by ScottDowne** [comment]
@edrushka @davidascher 
1. I think we can use a newsletter value for this. Right now we use "mozilla-foundation" I'm thinking thunderbird can use one like "mozilla-thunderbird"
2. I can ask this. I suspect we'll be fine, but I'm confirming.
3. @davidascher I'm not sure I follow.  It sounds like thunderbird needs double opt in? If so, yes, exact target is going to handle this, and I can confirm the exported list is only the double opt out confirmed emails.

**Comment by davidascher** [comment]
Yes I'm sure et will handle double optin (isn't that mandated by law?).
What I'm asking is if mailchimp can do a batch import of a CSV file or
whatever, of whether they need to manage the optin themselves. Should be
easy to figure out.

**Comment by ScottDowne** [comment]
@davidascher I'm not sure how optin and import are connected.

Also, I haven't heard back from the moco contact I got from @adamlofting for this, and it's possible it's not the right person currently.

Is there anywhere anyone can point me to who I need to contact to get this setup in exact target?

Once we have that contact, I can actually answer these questions regarding import. Right now my  guess is as good as yours.

**Comment by davidascher** [comment]
I may be paranoid, but I can imagine setups where an email as a service
company won't send an email to a list of email addresses if it doesn't have
logs and evidence that all of those folks have opted in.

So to answer this question I think just check out mailchimp docs to see if
there is an "import email list" feature.

Make sense?

**Comment by edrushka** [comment]
As for setting this up in ET, have you connected with Stephanie yet? She can probably help you get the ear of MoCo's Salesforce/ET manager.

**Comment by adamlofting** [comment]
FWIW Mailchimp et al do have a list import. The first time you do this they check your list against common spam lists, and ask you to complete a few verification steps and detail where the sign-ups come from - sometimes phoning you to check you're a real org etc.

But, based on this ticket: https://github.com/mozilla/donate.mozilla.org/issues/1289 can't we just stick a mailchimp sign-up form on the thank you page using that new Thunderbird account?

**Comment by davidascher** [comment]
Thanks Adam.

**Comment by ScottDowne** [comment]
@adamlofting that ticket you linked is helpful history. It seems a lot of what is a new topic to me is old news to everyone else :)

**Comment by ScottDowne** [comment]
So I'm totally fine with not using exact target after reading through #1289

**Comment by ScottDowne** [comment]
It seems like either way this is blocked, if we use exact target, we need an exact target bucket, if mailchimp, (or something else) vendor review and setup for that service needs to be done.

**Comment by edrushka** [comment]
@ScottDowne vendor review is underway #1345

See all TB tickets [here] (https://github.com/mozilla/donate.mozilla.org/issues?q=is%3Aissue+is%3Aopen+label%3AThunderbird).

**Comment by adamlofting** [comment]
We should check with legal, but as it wouldn't be Mozilla running the Mailchimp account it might not need our formal review process? We should however make sure the opt-in language is clear about where the data is going (both in terms of using Mailchimp and being owned by the Thunderbird team).

> It seems a lot of what is a new topic to me is old news to everyone else :)

FWIW I just found this now while trying to find Jess D's github handle. :four_leaf_clover:

**Comment by edrushka** [comment]
I'd be happy to schedule a quick circle up so we can all get on the same page. @ScottDowne are you taking over as dev for this entire TB project?

**Comment by ScottDowne** [comment]
@edrushka I would be fine with a call. Nothing official has me taking over dev for TB, I just jumped on this issue because it looked like no one else wanted it.

However, I would be fine finishing off anything else that's outstanding.

**Comment by edrushka** [comment]
@adamlofting are you interested in joining?

**Comment by adamlofting** [comment]
I'm probably more of a timezone-hinderence than a help for moving this along, so don't let me hold it up.

I'm happy to review any tickets if I can be helpful though.

**Comment by ScottDowne** [comment]
I don't think you're blocking, Adam. But don't be afraid to jump in. I can
always use info.

I'm going to add functionality for mail chimp. I don't see how having it
ready can be a bad thing.

We still have the option to use et temporarily while we wait for vendor
review.

**Comment by ScottDowne** [comment]
@edrushka I heard back from MoCo on potentially setting up a thunderbird basket in exact target, and they asked us to write a document that explains the following:

* Program goals and how program goals align with business goals
* Challenges and opportunities
* Your audience
*  Preliminary plans for content strategy and cadence

This seems like too much and too formal for a temporary place to put emails while we wait for vendor review, but maybe they'll be fine with our reasoning.

I'm also hoping I don't have to write that up :D

I asked for a sample and they sent me one a previous team wrote up to get an idea.

**Comment by edrushka** [comment]
![giphy](https://cloud.githubusercontent.com/assets/4515662/13647261/2750c8a0-e5e8-11e5-94e3-7c2363ef2fbd.gif)

Can you please forward me the sample @ScottDowne

**Comment by vazquez** [comment]
Best GIF of the day award goes to @edrushka! :trophy:
